### DISCUSSION TITLE
vcpkg Mac, M1, Arm, Intel, Universal package?

### DISCUSSION THREAD

**Discussion by Dariusz1989** [microsoft/vcpkg#24827] [Q&A] [answered]
Hey

As topic say, say I want to do ./vcpkg install zlib,
How can I install ARM64, Intel_x86 (I think)/universal package, etc so when I build my app I can target both intel/arm platforms?

Regards
Dariusz

**Comment by autoantwort** [comment, marked as answer]
See #19454. There is currently no solution provided by vcpkg. You have to build arm64 and x64 and merge the binaries yourself

**Reply by Dariusz1989** [reply]
I've intalled 2x vcpkg, 1 for each platform then target each one depending on arch target. I generate 2x binary at the end.

**Comment by AugP** [comment]
We are investigating this and have received multiple requests recently to add support for universal macOS binaries. I'll follow up with the team to find a good time to work on it and get it released.
